commands/utils/params.go

```diff
@@ -147,25 +148,44 @@ func extractGeneralParamsFromEnv(params *FrogbotParams) {
 func extractScanPullRequestParamsFromEnv(params *FrogbotParams) error {
 	includeAllVulnerabilities := getTrimmedEnv(IncludeAllVulnerabilitiesEnv)
```

Comment:
I think we can create a new function `getBoolEnv` (instead of `parseBoolEnv`) that gets and returns the default value if not provided. This way we can remove code duplication from `extractScanPullRequestParamsFromEnv`.

Suggested method signature:
```go
func getBoolEnv(envKey string, defaultValue bool) (bool, error)

Reply:
done